third_party/proto/ibc/lightclients/tendermint/v1/tendermint.proto

```diff
@@ -2,11 +2,11 @@ syntax = "proto3";
 
 package ibc.lightclients.tendermint.v1;
 
-option go_package = "github.com/cosmos/ibc-go/modules/light-clients/07-tendermint/types";
+option go_package = "github.com/cosmos/ibc-go/v3/modules/light-clients/07-tendermint/types";
 
 import "tendermint/types/validator.proto";
 import "tendermint/types/types.proto";
-import "confio/proofs.proto";
+import "proofs.proto";
```

Comment:
re othter comment; import path changed, so still need to incldue proofs in third_party